src/components/site/SummaryList.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+const SummaryList = props => {
+  const {
+    iconMarker,
```

Comment:
Icon needs to be flexible, as web services has the option to choose any of these https://fontawesome.com/icons?d=gallery. See above for the comment about making a `SummaryListItem` component. This make it easier to control the icon display.

Reply:
done